docs/website/content/docs/bootstrapping.md

```diff
@@ -84,72 +137,82 @@ EOF
 
 At this point, any servers on the same networkd as Sidero should PXE boot using the Sidero PXE service.
```

Comment:
from my testing it looks like servers are registered as soon as we boot Sidero components (before default environment is created)

Reply:
Yeah, that's true. it's not strictly required to precreate the default environment. I can move the ordering of those two steps.